_layouts/index.html

```diff
@@ -40,6 +40,7 @@ <h2 id="question">{{ t.pagecontent.question }}</h2>
           <p>{{ t.pagecontent.how }}</p>
         </div>
         <div class="col-2">
+          <div class="highlight-wrapper" tabindex="0">
```

Comment:
Don't need tab index on this or any of the ones below.

Reply:
Sorry I don't understand how do you scroll the "overflow-x: auto"?

Reply:
I cannot reproduce a situation in which they need to scroll?

Reply:
So is it that `overflow-x: auto` then can be removed? https://github.com/Homebrew/brew.sh/blob/master/assets/css/styles.scss#L207

Reply:
I tell a lie: no, it shouldn't be because then the `wget.rb` is trimmed. I still don't think a tab index is needed here.